apis/python/setup.py

```diff
@@ -151,7 +147,10 @@ def find_or_build_package_data(setuptools_cmd):
         # cause that cache to fall out of sync.
         #
         # See `.github/workflows/python-ci-single.yml` for configuration.
-        subprocess.run(["./bld"], cwd=scripts_dir)
+        if os.name == "nt":
+            subprocess.run(["pwsh.exe", "./bld.ps1"], cwd=scripts_dir, check=True)
```

Comment:
@beroy is currently working on removing the `bld` script and porting the cmake commands over into `setup.py`. I don't think anything needs to be modified in this particular PR but just letting relevant parties be aware.